templates/default.html

```diff
@@ -13,7 +13,14 @@
       </a>
       <nav role="navigation" aria-label="Main">
         <a role="menuitem" href="/download/">Download</a>
-        <a role="menuitem" href="/docs/">Documentation</a>
+        <div class="has-submenu">
+          <a role="menuitem" href="/docs/">Documentation</a>
+            <div class="submenu">
+              <a role="menuitem" href="/commands/">Command Reference</a>
+              <a role="menuitem" href="/docs/topics/">Documentation by Topic</a>
+              <a role="menuitem" href="/technical-blog-posts/">Technical Blog Posts</a>
+            </div>
+        </div>
         <a role="menuitem" href="/commands/">Commands</a>
```

Comment:
Can we remove commands reference here since its under oDocumentation header now?

Reply:
Yeah, decent idea.